rivery_cli/cli/activities.py

```diff
@@ -83,13 +89,34 @@ def download_run_logs(ctx, **kwargs):
     click.echo(
         f'Run ID {run_id} logs fetched successfully.'
     )
+
+    logs = str(resp.content)
```

Comment:
Does the resp.content returned as json/csv format?

Reply:
the actual string is in csv format, but the content itself is a byte array